src/routes/safe/components/Apps/index.tsx

```diff
@@ -37,6 +37,10 @@ const Centered = styled.div`
   align-items: center;
   justify-content: center;
   flex-direction: column;
+
+  &.mt {
```

Comment:
What I meant was actually:
```
const CenteredMt = styled(Centered)`
  margin-top: 5px;
`
```
or 
```
const Centered = styled.div<{ marginTop }>`
  ...
  margin-top: props.marginTop || 0;
`
```
This way the className `mt` is not transpiled/obfuscated and could lead to possible conflicts

Reply:
done!